tests/test_helpers.py

```
@@ -45,6 +109,33 @@ def create_file(path: str, content: str = "", sleep_time: float = 0.1) -> None:
     time.sleep(sleep_time)
 
 
+def cleanup_test_dirs(base_dir: str, dirnames: list, logger=None) -> None:
```

Ruff (0.13.3)

Redefinition of unused `cleanup_test_dirs` from line 59

(F811)